src/main/java/com/quiz/service/impl/QuizServiceImpl.java

```diff
@@ -1,6 +1,129 @@
 package com.quiz.service.impl;
 
+import com.quiz.config.DBConnectorConfig;
+import com.quiz.model.QuizQuestion;
 import com.quiz.service.QuizService;
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.UUID;
+
 public class QuizServiceImpl implements QuizService {
+    @Override
+    public void addQuizQuestion(Long quizId, String question) {
```

Comment:
why did you write this in ServiceImpl class? isn't it exist in QuizQuestionRepository? can you explain thank you